ugt_fwtools/build_report.py

```diff
@@ -8,7 +8,7 @@
 import os
 from datetime import datetime
 
-ALL_FORMATS = ["markdown", "textile"]
+ALL_FORMATS = ["markdown", "textfile"]
```

Comment:
**Note:** key `textile` is not a type but refers to the [textile](https://textile-lang.com/) markup language.

Reply:
fixed in 8a20538

Reply:
ok